hudi-common/src/main/java/org/apache/hudi/common/table/view/TableFileSystemView.java

```diff
@@ -148,6 +148,9 @@
    */
   Stream<HoodieFileGroup> getAllFileGroups(String partitionPath);
 
+  Stream<String> getAllExcludeFileGroups(String partitionPath);
```

Comment:
I think I have mentioned it somewhere else. Lets denote this feature with the consistent term "Replace" everywhere.

Reply:
Done